test/clj/nnts2/note/functional_test.clj

```diff
@@ -1,41 +1,46 @@
 (ns nnts2.note.functional-test
-  (:require  [clojure.test :refer :all]
-             [nnts2.fixtures :as fixtures]
-             [nnts2.handler.note :refer [create]]
-             [nnts2.db.note :refer [add]]))
-
+  (:require [clojure.test :refer :all]
+            [nnts2.fixtures :as fixtures]
+            [nnts2.handler.note :refer [create]]
+            [nnts2.db.note :refer [add]])
+  (:import (org.postgresql.util PSQLException)
+           (java.util UUID)))
 
 (use-fixtures :each fixtures/clear)
 (use-fixtures :once fixtures/setup fixtures/adduser)
 
-
 (deftest create-note-success
   (testing "Should create note successfully for valid params"
-    (let [request {:nnts-user 1 :body {:title "note-title" :content "note-content"}}
+    (let [request {:nnts-user (UUID/fromString "820bb852-445f-4101-b257-f84f66aa74ff")
```

Comment:
for now, define the user uuid in the let for deftest and use it in testing blocks. Its repetitive now